### DISCUSSION TITLE
vim_stuff/search-and-replace/

### DISCUSSION THREAD

**Discussion by giscus** [Jitesh117/blog_comments#16] [Q&A]
# vim_stuff/search-and-replace/

Search and replace is a powerful feature in vim that can save you a lot of time when editing text. Whether you’re working on a single file or across multiple files, vim provides robust tools to find and replace text quickly. In this article, we’ll explore the basics of search and replace, as well as some advanced tips to help you make the most of this feature.
But why would one use this when LSPs can do the same thing but better?

https://jitesh117.github.io/vim_stuff/search-and-replace/

**Comment by hyperdevstuff** [comment]
Didn't knew the last two advanced f&r thanks G !
